[Docs] 建议优化readme中Deploymen内容

### 📚 The doc issue


1. InternLM仓deployment跟LMDeploy仓部署步骤差别较多

<img width="1201" alt="企业微信截图_16903406865826" src="https://github.com/InternLM/InternLM/assets/126147887/d1c88231-2e8e-41ba-8183-d9071ab92347">

2. 另外 “ python3 -m lmdeploy.serve.turbomind.deploy **InternLM-7B** /path/to/internlm-7b/model hf” 还有描述错误
   
“InternLM-7B”应该为“internlm-chat-7b”
按照原文档中命令会有如下报错：
assert model_name in MODELS.module_dict.keys(), \
AssertionError: 'InternLM-7B' is not supported. The supported models are: dict_keys(['llama', 'vicuna', 'internlm', 'internlm-chat-7b', 'internlm-chat-7b-8k', 'puyu', 'baichuan-7b', 'llama2'])

--- a/README-zh-Hans.md
+++ b/README-zh-Hans.md
@@ -119,21 +119,22 @@ streamlit run web_demo.py
 
 1. 首先安装 LMDeploy:
 
-  ```
-  python3 -m pip install lmdeploy
-  ```
+   ```bash
+   python3 -m pip install lmdeploy
+   ```
 
 2. 快速的部署命令如下：
 
-  ```
-  python3 -m lmdeploy.serve.turbomind.deploy internlm-chat-7b /path/to/internlm-7b/model
-  ```
+   ```bash
+   python3 -m lmdeploy.serve.turbomind.deploy internlm-chat-7b /path/to/internlm-7b/model
+   ```
 
-3. 在导出模型后，你可以直接通过如下命令启动服务一个服务并和部署后的模型对话
+3. 在导出模型后，你可以直接通过如下命令启动服务，并在客户端与AI对话
 
-  ```
-  python3 -m lmdeploy.serve.client {server_ip_addresss}:33337
-  ```
+   ```bash
+   bash workspace/service_docker_up.sh
+   python3 -m lmdeploy.serve.client {server_ip_addresss}:33337
+   ```
 
 [LMDeploy](https://github.com/InternLM/LMDeploy) 支持了 InternLM 部署的完整流程，请参考 [部署教程](https://github.com/InternLM/LMDeploy) 了解 InternLM 的更多部署细节。
 

--- a/README-zh-Hans.md
+++ b/README-zh-Hans.md
@@ -126,7 +126,7 @@ streamlit run web_demo.py
 2. 快速的部署命令如下：
 
   ```
-  python3 -m lmdeploy.serve.turbomind.deploy InternLM-7B /path/to/internlm-7b/model hf
+  python3 -m lmdeploy.serve.turbomind.deploy internlm-chat-7b /path/to/internlm-7b/model
   ```
 
 3. 在导出模型后，你可以直接通过如下命令启动服务一个服务并和部署后的模型对话

--- a/README.md
+++ b/README.md
@@ -126,21 +126,22 @@ We use [LMDeploy](https://github.com/InternLM/LMDeploy) to complete the one-clic
 
 1. First, install LMDeploy:
 
-```
-  python3 -m pip install lmdeploy
-```
+    ```bash
+    python3 -m pip install lmdeploy
+    ```
 
 2. Use the following command for quick deployment:
 
-```
-  python3 -m lmdeploy.serve.turbomind.deploy internlm-chat-7b /path/to/internlm-chat-7b/model
-```
+    ```bash
+    python3 -m lmdeploy.serve.turbomind.deploy internlm-chat-7b /path/to/internlm-chat-7b/model
+    ```
 
 3. After exporting the model, you can start a server and have a conversation with the deployed model using the following command:
-
-```
-  python3 -m lmdeploy.serve.client {server_ip_addresss}:33337
-```
+   
+    ```bash
+    bash workspace/service_docker_up.sh
+    python3 -m lmdeploy.serve.client {server_ip_addresss}:33337
+    ```
 
 [LMDeploy](https://github.com/InternLM/LMDeploy) provides a complete workflow for deploying InternLM. Please refer to the [deployment tutorial](https://github.com/InternLM/LMDeploy) for more details on deploying InternLM.
 

--- a/README.md
+++ b/README.md
@@ -133,7 +133,7 @@ We use [LMDeploy](https://github.com/InternLM/LMDeploy) to complete the one-clic
 2. Use the following command for quick deployment:
 
 ```
-  python3 -m lmdeploy.serve.turbomind.deploy InternLM-7B /path/to/internlm-7b/model hf
+  python3 -m lmdeploy.serve.turbomind.deploy internlm-chat-7b /path/to/internlm-chat-7b/model
 ```
 
 3. After exporting the model, you can start a server and have a conversation with the deployed model using the following command:

